projects/storefrontlib/src/shared/components/anonymous-consents-dialog/anonymous-consent-dialog.component.ts

```diff
@@ -45,7 +75,7 @@ export class AnonymousConsentDialogComponent implements OnInit, OnDestroy {
   }
 
   closeModal(reason?: any): void {
-    this.modalService.closeActiveModal(reason);
+    this.closeDialog.emit(reason);
```

Comment:
I'm not super exited about how the 2 components are decoupled. I'd like to either make a generic approach to focus the caller element in the launch service, or leave it up to the specific caller/dialog. The latter is less complex and a good start I'd say. What about implementing a close subject in the dialog service that the caller component can observe?

Reply:
Added a subject to the service